photo-app/app/models/payment.rb

```diff
@@ -0,0 +1,19 @@
+class Payment < ApplicationRecord
+  attr_accessor :card_number, :card_cvv, :card_expires_month, :card_expires_year
+  belongs_to :user
+  def self.month_options
```

Comment:
What do you mean by `month_options`?

Reply:
`self.month_options`, `self.year_options` are used in the sign-up form for the users to select the month and year. The method `self.month_options` returns following to use in the form

```
[["1 - January", 1],
 ["2 - February", 2],
 ["3 - March", 3],
 ["4 - April", 4],
 ["5 - May", 5],
 ["6 - June", 6],
 ["7 - July", 7],
 ["8 - August", 8],
 ["9 - September", 9],
 ["10 - October", 10],
 ["11 - November", 11],
 ["12 - December", 12]]
```

`self.year_options` returns:
`[2022, 2023, 2024, 2025, 2026, 2027, 2028] `

Reply:
My point with this question is that the naming is not clear

Reply:
Not planning to change the naming?